### DISCUSSION TITLE
[Feature Request] Group and introduce comments

### DISCUSSION THREAD

**Discussion by yusancky** [community/community#27263] [Other Feature Feedback, Questions, & Ideas]
Sometimes, we may comment on a same topic (such as how to modify a piece of content) on a pull request's Conversation tab. After the topic has been discussed, pages tend to appear cumbersome. 
However, if we can introduce these conversations in one sentence (such as `We choose to ...` ), others can get a quick overview of what we're talking about. If GitHub will add this feature, the efficiency will be improved to some extent. 

After setting the introductory sentences, the comments can be displayed like a card with `Load more...` _(not drawn in the figure)_, as shown in the following figure. _You can also [see the detailed picture there](https://github.com/community/community/discussions/27263#discussioncomment-3279222)._ 

![76DB8A1E-B1B3-4B18-A2D6-E30FC60BE109](https://user-images.githubusercontent.com/83524927/181694235-e7b4e5b6-99e9-4c17-9978-e3d012a43337.jpeg)

**Comment by jfullstackdev** [comment]
can you compare it from the current pull request view that you are referring too ? and can you make that edited proposed view a true example like there are actual names and topic, so we can see the difference

**Reply by yusancky** [reply]
> can you compare it from the current pull request view that you are referring too ? and can you make that edited proposed view a true example like there are actual names and topic, so we can see the difference

@jdevstatic For example, the following picture _(sorry but but it uses Chinese Simplified)_ shows how we discussed **whether we could merge the PR before writing further (Topic)** _(We have written a wiki but the author of the PR was temporarily out of response)_.  
![image](https://user-images.githubusercontent.com/83524927/181734923-d503e594-639a-4df2-82aa-17a5a57ff214.png)

After setting the topic by somebody who has write access, the card will be shown like the following image. 
![image](https://user-images.githubusercontent.com/83524927/181737909-2fbff23e-af97-4d19-99a7-fbe9c1dcebbc.png)
